### DISCUSSION TITLE
Associate one Contact to more Companies

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14812] [How to]
Hi,
I would assign the same contact to more companies.
It is possible?
@mariuszkrzaczkowski

**Comment by bpabiszczak** [comment]
We set up such a relationship with several clients. We do not recommend this.

**Reply by espe808** [reply]
And so, how could I proceed?

**Reply by espe808** [reply]
The need is:
I have some professionals (lawyers, engineers...) that could be associated to several companies and so this is the reason of my questions. @mariuszkrzaczkowski @bpabiszczak

**Reply by mariuszkrzaczkowski** [reply]
In your case it will be an M: M relation and I wouldn't change the existing one because it utilizes a different business process. Add an additional M: M relation (not the reference field). It is the simplest and least painful for the system logic.
